gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/package-info.java

```diff
@@ -32,6 +34,21 @@
  * RemoteGcsHelper.forceDelete(storage, bucket, 5, TimeUnit.SECONDS);
  * } </pre>
  *
+ * LocalGcsHelper helps with testing on an in-memory filesystem (this is best for unit tests).
```

Comment:
Maybe this documentation should be moved into the classes themselves, and the package-info can use @link tags to reference them.